### PR TITLE
Add a notice before staking to top 20 validators to encourage decentralization

### PR DESCRIPTION
**Pull request by lukaw3d**
Mitigates https://github.com/oasisprotocol/oasis-wallet-web/issues/766

notice

### CODE REVIEW

File: src/app/components/AddEscrowForm/__tests__/index.test.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ const renderComponent = (store: any, address: string, validatorStatus: Validator
   render(
     <Provider store={store}>
       <ModalProvider>
-        <AddEscrowForm validatorAddress={address} validatorStatus={validatorStatus} />
+        <AddEscrowForm validatorAddress={address} validatorStatus={validatorStatus} validatorRank={30} />
```
Comment (aefhm) [inline src/app/components/AddEscrowForm/__tests__/index.test.tsx:15]: `30` or `21` for edge testing?

File: src/app/components/AddEscrowForm/index.tsx [hunk 2]
```diff
@@ -61,30 +64,47 @@ export const AddEscrowForm = memo((props: Props) => {
...
-            min={minStaking}
-            value={amount}
-            onChange={event => setAmount(event.target.value)}
-            required
+    <>
+      {showNotice && (
+        <p>
+          {t(
+            'account.addEscrow.confirmDelegatingToTop.description',
+            'This validator is ranked in the top 20 by stake. Please consider delegating to a smaller validator to increase network security and decentralization.',
```
Comment (aefhm) [inline src/app/components/AddEscrowForm/index.tsx:72]: I guess more of a copy question for product, but my first instinct is the softer approach of prodding the user through giving them context.

File: src/app/components/AddEscrowForm/index.tsx [hunk 3]
```diff
@@ -61,30 +64,47 @@ export const AddEscrowForm = memo((props: Props) => {
...
+          )}
+          <CheckBox
+            label={t('account.addEscrow.confirmDelegatingToTop.acknowledge', "I don't want to!")}
```
Comment (Yawning) [inline src/app/components/AddEscrowForm/index.tsx:75]: "Delegate anyway" or something similar (more professional) perhaps.

File: src/app/components/AddEscrowForm/__tests__/index.test.tsx (hunk 1 above)
Comment (lukaw3d) [reply src/app/components/AddEscrowForm/__tests__/index.test.tsx]: :shrug: changed

File: src/app/components/AddEscrowForm/index.tsx (hunk 2 above)
Comment (lukaw3d) [reply src/app/components/AddEscrowForm/index.tsx:72]: @peterjgilbert

Comment (lukaw3d) [reply src/app/components/AddEscrowForm/index.tsx:72]: i think it's fine :D

**Review by tjanez — APPROVED** [review]

**Pull request merged**